src/devices/is42s32400f.rs

```diff
@@ -0,0 +1,57 @@
+/// ISI IS42S32400F SDRAM
+#[allow(unused)]
+
+/// Speed Grade 6
+pub mod is42s32400g_6 {
```

Comment:
```suggestion
pub mod is42s32400f_6 {
```
I had added this same memory before seeing your PR; the only difference being the module name which I think should include `f` instead of `g`.  Test code in https://gitlab.com/mfairman/stm32f469i-disc/-/tree/mfairman/sdram under `examples/fmc_sdram_test.rs`

It would be great to get this merged and released so the updated crate can be used directly.